app/src/main/kotlin/at/bitfire/davdroid/sync/worker/SyncWorkerManager.kt

```diff
@@ -115,12 +114,19 @@ class SyncWorkerManager @Inject constructor(
     /**
      * Requests immediate synchronization of an account with a specific authority.
      *
-     * @param account               account to sync
-     * @param dataType              type of data to synchronize
-     * @param manual                user-initiated sync (ignores network checks)
-     * @param resync                whether to request (full) re-synchronization (`null` for normal sync)
-     * @param syncFrameworkUpload   see [ContentResolver.SYNC_EXTRAS_UPLOAD] – only used for contacts sync and Android 7 workaround
-     * @param fromPush              whether this sync is initiated by a push notification
+     * If there's already a one-time sync running, another sync is appended to make sure a complete sync is run
+     * since the moment this method is called.
+     *
+     * Otherwise (no currently running one-time sync), no new sync is appended, so this method just lets the current sync continue.
```

Comment:
Should probably say one of these two instead? 

Otherwise (no currently running one-time sync), a new sync request is enqueued.

Otherwise (if a sync is running and there is already an appended sync), no new sync is appended, so this method just lets the current sync continue and does not append more.